src/main/java/com/tsunderebug/speedrun4j/exception/SpeedrunJsonException.java

```diff
@@ -0,0 +1,9 @@
+package com.tsunderebug.speedrun4j.exception;
```

Comment:
If the major version is being upgraded, it might be worth it to update the domain from `com.tsunderebug` to `tf.bug` to reflect my other works.